pkg/controllers/management/auth/project_cluster_handler.go

```diff
@@ -271,36 +264,30 @@ type mgr struct {
 }
 
 func (m *mgr) createDefaultProject(obj runtime.Object) (runtime.Object, error) {
-	return m.createProject(project.Default, v32.ClusterConditionconditionDefaultProjectCreated, obj, defaultProjectLabels, defaultProjects)
```

Comment:
> Currently in v2.5.9, the system and default projects will not be created
in the hosted Rancher setup

If the project is not created, then what causes this condition `ClusterConditionconditionDefaultProjectCreated` to be true?

Reply:
In 2.5.9 and in the hosted Rancher setup, the local cluster won't have a `creatorID`. So this code will set the condition to `true`: https://github.com/rancher/rancher/blob/release/v2.5.9/pkg/controllers/management/auth/project_cluster_handler.go#L269-L273